app/controllers/concerns/api/v2/taxonomies_controller_extension.rb

```diff
@@ -0,0 +1,12 @@
+module Api::V2::TaxonomiesControllerExtension
```

Comment:
how about changing this to be more informative? like `TaxonomiesControllerDocExtension` ?